src/main/java/com/jcraft/jsch/ChannelSftp.java

```diff
@@ -158,6 +161,7 @@ public class ChannelSftp extends ChannelSession {
   private boolean fEncoding_is_utf8 = true;
 
   private RequestQueue rq = new RequestQueue(16);
+  private String chunkStart = null;
```

Comment:
Why does `chunkStart` need to be a `String` instead of having it simply be a `long`?
This would avoid a lot of unwieldy calls to `Long.parseLong(chunkStart)`?

Reply:
Changed it to Long now and made the other changes accordingly.